.maestro/setup_ui_tests.sh

```diff
@@ -0,0 +1,112 @@
+### Set up environment for UI testing
+
+## Constants
+derived_data_path=DerivedData
+app_location=$derived_data_path/Build/Products/Debug-iphonesimulator/DuckDuckGo.app
+device_uuid_path="$derived_data_path/device_uuid.txt"
+
+# The simulator command requires the hyphens
+target_device="iPhone-15"
+target_os="iOS-17-2"
+
+## Functions
+
+fail() {
+    echo "‼️ $1"
+    exit 1
+}
+
+check_is_root() {
+    if [ ! -d ".maestro" ]; then
```

Comment:
I can make this fail, for example by invoking from a parent folder, but it works in the most common scenarios for sure. 
Checking it in a more robust way is much more verbose, so I'm inclined to say this is good enough. 
But what's the worst thing that can happen if this is invoked from other directories?

Reply:
Hmm, not much tbh, it'll fail because it can't build the app, but wanted to avoid people having to work that out.